src/pages/GameDetailPage.tsx

```diff
@@ -16,13 +16,17 @@ const GameDetailPage = () => {
   if (error || !game) throw error;
 
   return (
-    <>
-      <Heading>{game.name}</Heading>
-      <ExpandableText>{game.description_raw}</ExpandableText>
-      <GameAttributes game={game} />
-      <GameTrailer gameId={game.id} />
-      <GameScreenshots gameId={game.id} />
-    </>
+    <SimpleGrid columns={{ sm: 1, md: 2 }} spacing={5}>
+      <GridItem>
```

Comment:
feat: can be added AI recommendation system in future.

Reply:
+